nilearn/reporting/glm_reporter.py

```diff
@@ -907,14 +907,6 @@ def _stat_map_to_png(
             cmap = "Blues_r"
```

Comment:
This code is enough for plotting to view data thresholded with negative threshold value and `two_sided=False`